R/add_panel.R

```diff
@@ -5,8 +5,13 @@
 #' @param session shiny session object.
 #' See \url{https://dockview.dev/docs/api/dockview/panelApi}.
 #' @export
-add_panel <- function(proxy, panel, ..., session = shiny::getDefaultReactiveDomain()) {
-  if (panel$id %in% list_panels(proxy, session)) stop("The panel ID you used is already in use!")
-  session$sendCustomMessage("add-panel", panel)
+add_panel <- function(
+  proxy,
+  panel,
+  ...,
+  session = shiny::getDefaultReactiveDomain()
+) {
+  if (panel$id %in% list_panels(proxy, session))
+    stop("The panel ID you used is already in use!")
+  session$sendCustomMessage(sprintf("%s_add-panel", proxy), panel)
```

Comment:
@nelstevens I realised that we did not prefix the message id by the proxy id which means, having 2 dock_view instances in the same app, they would interfere since messages are intercepted in the same channel (by the same handler).